keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -258,11 +274,25 @@ Introduce e2e test that run both with etcd progress notify events enabled
 and disable to ensure both configurations work correctly (both with this
 feature enabled and disabled)
 
-Benchmark consistent reads from cache against consistent reads from etcd for:
-- list result sizes of 1, 10, ..., 100000
-- object sizes of 5kb, 25kb, 100kb
-- measure latency and throughput
-- document results in this KEP
+Benchmark consistent reads from cache against consistent reads.
+Added performance tests to https://testgrid.k8s.io/sig-scalability-experiments, following 2 scenarios:
+* Small objects - 300`000 configmaps each 1KB of size. 
+* Large objects - 300 configmaps each 1MB of size.
+
+In both cases we put load of 1 LIST per second with empty label selector.
```

Comment:
This is misleading - I think your selector is selecting "no objects", (whereas empty selector is selecting everything).

Reply:
True, fixed.